stable/nginx-ingress/templates/controller-service.yaml

```diff
@@ -0,0 +1,29 @@
+{{- $root := . -}}
+{{- range .Values.controller.services }}
+---
```

Comment:
Tests are failing because of this. Do we have to loop in this file? When would I have multiple? Can we start with this hardcoded to 1 service?